src/babel-plugin-transform-sx-tailwind/src/index.js

```diff
@@ -62,6 +67,7 @@ module.exports = function sxTailwindBabelPlugin(): any {
       },
       Program: {
         enter(path, state) {
+          stylesCollector = [];
```

Comment:
👍 I think this was the main culprit. Is there a simple way how to test it?

Reply:
Exactly, it was collecting styles incrementally from all files. Unfortunately no idea how to test it :-(